cmake/libmultiprocess.cmake

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2025 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://opensource.org/license/mit/.
+
+function(add_libmultiprocess subdir)
+  # Unconditionally set BUILD_TESTING to ON. BUILD_TESTING is a standard cmake
+  # variable that controls whether CTest is used. In the bitcoin build, CTest is
+  # always used and we do not want an option to turn it off. But in
+  # libmultiprocess, CTest is optional and BUILD_TESTING needs to be set in
+  # order to add libmultiprocess unit tests to the CTest configuration.
+  set(BUILD_TESTING ON)
```

Comment:
88eabc8c2c0fbb62db5cb6248cc8ab770f3ce78b

I apologies for being pedantic, but the sentence _"BUILD_TESTING is a standard cmake variable that controls whether CTest is used."_ is not 100% correct, as the `BUILD_TESTING` variable is [_created_](https://cmake.org/cmake/help/latest/module/CTest.html) by the `CTest` module. However, I'm not sure how to reword the comment best.

Reply:
> I was surprised to see bitcoin build is not calling `enable_testing()` when `BUILD_TESTS` is off...

Why? What alternative design would you suggest?